how-to-build.md

```diff
@@ -56,19 +56,27 @@ to download the appropriate installers.
 
 ## Configure, build & test
 
+To get submodule dependencies:
+    
+    git submodule update --init --recursive
+    
 Run the following commands to configure, build and test core:
 
     mkdir build-dir # create a build folder
     cd build-dir
```

Comment:
I think we should recommend that you create a directory specific for the build type. So for debug it would be
`mkdir build.debug; cd build.debug`. This will also be compatible with the eclipse project files.